### DISCUSSION TITLE
How to set up remote model's API Key?

### DISCUSSION THREAD

**Discussion by nirajgiriXD** [janhq/jan#2621] [Get Help] [answered]
I started a thread in Jan but it is showing `invalid API key`, where can I get the API key for Jan?

![image](https://github.com/janhq/jan/assets/51747980/afed1479-130e-470e-80a9-26e401e4611d)

**Comment by ux-han** [comment, marked as answer]
@nirajgiriXD What models do you want to use? You can set up API keys in the extension settings page in the latest Jan's release. 

<img width="1104" alt="Screenshot 2024-09-02 at 8 25 56 PM" src="https://github.com/user-attachments/assets/ba73c3bb-1f19-41cb-afa3-6d7a2c86428a">
